set.mm

```diff
@@ -26071,7 +26072,8 @@ choice between (what we call) a "definitional form" where the shorter
     $( Consequence of the not-free predicate.  (Contributed by Mario Carneiro,
        11-Aug-2016.) $)
     nfcrii $p |- ( y e. A -> A. x y e. A ) $=
-      ( vz cv wcel nfcriv nf5ri hblem ) AEBCEFCGAAECDHIJ $.
+      ( vz cv wcel wsb wal nfcriv nfsbv nf5ri clelsb3 albii 3imtr3i ) EFCGZEBHZ
```

Comment:
so mainly, nfsb (used in the proof of hblem<-hbsb<-nfsb) was replaced by nfsbv. Is it worth adding hbsbv ? Not sure (then, there would be many more (look at usages of hbsb recursively), and maybe not worth adding that many lemmas).

Reply:
Theorem `hbsbv` would need to have x, y and x, z as additional disjoint variable conditions, so it would look like this:
<pre>
 ${
   $d x y z $.
   hbsbv.1 $e |- ( ph -> A. z ph ) $.
   $( Version of ~ hbsb with additional disjoint variable conditions requiring
      fewer axioms.  (Contributed by Gino Giotto, 11-Aug-2023.) $)
   hbsbv $p |- ( [ y / x ] ph -> A. z [ y / x ] ph ) $=
     ( wsb nf5i nfsbv nf5ri ) ABCFDABCDADEGHI $.
 $}
</pre>

If `hbsbv` would be used in the proof of `nfcrii` the size would be the same as the current change:
 <pre>
$= ( vz cv wcel wsb wal nfcriv nf5ri hbsbv clelsb3 albii 3imtr3i ) EFCGZEBHZQAIBFCGZRAIPEBAPAAECDJKLBECMZQRASNO $.
$= ( vz cv wcel wsb wal nfcriv nfsbv nf5ri clelsb3 albii 3imtr3i ) EFCGZEBHZQAIBFCGZRAIQAPEBAAECDJKLBECMZQRASNO $.
</pre>

So in this case it doesn't offer any clear advantage. It might be useful for other theorems, but I have to do a scan for it.

Reply:
Theorem `hbsbv` would need to have x, y and x, z as additional disjoint variable conditions, so it would look like this:
<pre>
 ${
   $d x y z $.
   hbsbv.1 $e |- ( ph -> A. z ph ) $.
   $( Version of ~ hbsb with additional disjoint variable conditions requiring
      fewer axioms.  (Contributed by Gino Giotto, 11-Aug-2023.) $)
   hbsbv $p |- ( [ y / x ] ph -> A. z [ y / x ] ph ) $=
     ( wsb nf5i nfsbv nf5ri ) ABCFDABCDADEGHI $.
 $}
</pre>

If `hbsbv` would be used in the proof of `nfcrii` the size would be the same as the current change:
 <pre>
( vz cv wcel wsb wal nfcriv nf5ri hbsbv clelsb3 albii 3imtr3i ) EFCGZEBHZQAIBFCGZRAIPEBAPAAECDJKLBECMZQRASNO $.
( vz cv wcel wsb wal nfcriv nfsbv nf5ri clelsb3 albii 3imtr3i ) EFCGZEBHZQAIBFCGZRAIQAPEBAAECDJKLBECMZQRASNO $.
</pre>

So in this case it doesn't offer any clear advantage. It might be useful for other theorems, but I have to do a scan for it.

Reply:
Thanks @GinoGiotto. Do as you think is best.